sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdClientChannelPool.java

```diff
@@ -936,6 +1026,7 @@ private boolean offerChannel(final Channel channel) {
      * @see #acquire(Promise)
      */
     private Channel pollChannel() {
```

Comment:
>pollChannel [](start = 20, length = 11)

I like the 'safe' naming convention used for new code. 
Worth adding that prefix for the same for other existing safe methods as well.